Maman11/Q1/my_add.c

```diff
@@ -44,7 +29,7 @@ int decimalToBinary(int num){
     printf("Binary representation: %s\n", binary);
```

Comment:
SOLID - Read about SOLID

SOLID is:
Single Responsibility Principle (SRP)
Open-Closed Principle (OCP)
Liskov Substitution Principle (LSP)
Interface Segregation Principle (ISP)
Dependency Inversion Principle (DIP)


You dont need to understand everything at the moment and this is more a principle for OOP
but the important thing is S =  Single Responsibility Principle (SRP)
Each pice of code you write should be a single responsibility.
In this function you Both convert a number to binary and print the number. This is wrong
From solid we understand that each function hsould have only one responsibility thus this function should one hae one responsibility